src/notifications/warning-message.js

```diff
@@ -14,6 +14,7 @@ export default {
     maximizeError:                           'Was unable to maximize the window due to an error.\n\n{errMessage}',
     requestMockCORSValidationFailed:         '{RequestHook}: CORS validation failed for a request specified as {requestFilterRule}',
     debugInHeadlessError:                    'You cannot debug in headless mode.',
+    cannotFindWindowDescriptorError:         'An error has occurred while finding the window descriptor.\n\n{errMessage}',
```

Comment:
```suggestion
    cannotFindWindowDescriptorError:         'An error has occurred while searching for a window descriptor.\n\n{errMessage}',
```

Also we need to hear what @VasilyStrelyaev thinks about it.